pkg/server/server.go

```diff
@@ -185,6 +185,7 @@ func CredentialAPI(rg *gin.RouterGroup, service svcframework.Service, webhookSer
 	// Credentials
 	credentialAPI := rg.Group(CredentialsPrefix)
 	credentialAPI.PUT("", middleware.Webhook(webhookService, webhook.Credential, webhook.Create), credRouter.CreateCredential)
+	credentialAPI.PUT("/batchCreate", middleware.Webhook(webhookService, webhook.Credential, webhook.BatchCreate), credRouter.BatchCreateCredentials)
```

Comment:
This will fire once and not 1000 times right?
the output may be too large for a http POST req... 

also I wonder if someone is just listenting to /CreateCredential if it will fire 1000 times

Reply:
Seems like only if they subscribe to BatchCreate it will fire and not Create-Credential

But yea the post with the data will be too large.

Speaking of that, if we really make an integ test for 1000 creds the golang part may fail, like we may have a max of 10mb per response default config somewhere or something, worth testing!